root/layout/components/sidebar/SidebarAcousticBrainz.js

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
One more thing is that we'll need to move this file to somewhere under root/static/scripts/common/components/, perhaps root/static/scripts/common/components/sidebar/SidebarAcousticBrainz.js. This is mentioned briefly in HACKING.md, but basically scripts that run on the client (even if they're also used on the server) generally have to go under root/static/scripts/ so that translated strings work. The reason is that when we bundle translated strings as .js for the client, we want to only include strings actually used by client scripts to minimize the bundle size, as there are a lot of strings in server-side components that we don't need in the browser. So, we only extract client strings from root/static/scripts/.

Reply:
I moved the file to root/static/scripts/common/components/sidebar/
And I should definitely read again HACKING.md, that's on me
Thanks for the explanation